src/stylesheets/layouts/_navigation.scss

```diff
@@ -0,0 +1,38 @@
+.nav {
```

Comment:
oh! now we have the nav code in here too. Having issues with branching? Happy to help!

Reply:
I understand this isn't done, so I won't comment on the `nav` bits yet.

Reply:
ok cool . * work in progress *